source/tyr/tyr/resources.py

```diff
@@ -550,3 +567,105 @@ def delete(self, id):
             logging.exception("fail")
             raise
         return ({}, 204)
+
+class TravelerProfile(flask_restful.Resource):
+    """
+    Traveler profile api for creating updating and removing
+    """
+    @staticmethod
+    def get_args():
+        parser = reqparse.RequestParser()
+        parser.add_argument('coverage', type=str, required=True, case_sensitive=False)
+        parser.add_argument('traveler_type', type=parser_args_type.option_value(acceptable_traveler_types), required=True, case_sensitive=False)
+        parser.add_argument('walking_speed', type=parser_args_type.float_gt_0, required=False, case_sensitive=False)
+        parser.add_argument('bike_speed', type=parser_args_type.float_gt_0, required=False, case_sensitive=False)
```

Comment:
you can specify where to look for the values, I like to allow json as well with:
```
location=('json', 'values')
```